vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -724,7 +726,7 @@ void LedgerImpl::OnTimer(uint32_t timer_id) {
 
   if (timer_id == last_grant_check_timer_id_) {
     last_grant_check_timer_id_ = 0;
-    FetchGrants(std::string(), std::string(),
+    FetchGrants(std::string(), std::string(), std::string(),
```

Comment:
prefer `""` over `std::string()`

Reply:
There are lots of places in the code where `std::string()` is used. Why it became a problem in this PR?

Reply:
It was agreed a while ago with rewards team that `std::string()` was not a good choice and `""` has been used for all new code

Reply:
we need to clean up this places as we agreed to do `""` everywhere

Reply:
If we are going that way, I would like to see a document that describes all that requirements for the code. Put it in the repo, put it in the wiki.